### PR TITLE
CEDS-1673 Remove unnecessary PUT

### PR DESCRIPTION
**Pull request by ed0906**
- Remove WCO library as its no longer used
- Remove Request from service layer

### CODE REVIEW

File: app/controllers/declaration/SummaryController.scala [hunk 1]
```diff
@@ -70,7 +70,7 @@ class SummaryController @Inject()(
             BadRequest(summaryPage(Mode.Normal, SupplementaryDeclarationData(request.cacheModel), formWithErrors))
         ),
         legalDeclaration => {
-          submissionService.submit(request.cacheModel, legalDeclaration).map {
+          submissionService.submit(request.authenticatedRequest.user.eori, request.cacheModel, legalDeclaration).map {
```
Comment (PatrykRudnicki) [inline app/controllers/declaration/SummaryController.scala]: I'm not sure but we should have `eori` field in `JourneyRequest` so you don't need to go through `authenticatedRequest.user.eori`

Comment (ed0906) [reply app/controllers/declaration/SummaryController.scala]: It wasn't on the request but it is now 👍

File: app/models/requests/JourneyRequest.scala [hunk 2]
```diff
@@ -23,4 +23,5 @@ import play.api.mvc.WrappedRequest
 class JourneyRequest[A](val authenticatedRequest: AuthenticatedRequest[A], val cacheModel: ExportsDeclaration)
     extends AuthenticatedRequest[A](authenticatedRequest, authenticatedRequest.user) {
   val choice: Choice = Choice(cacheModel.choice)
+  def eori: String = authenticatedRequest.user.eori
```
Comment (PatrykRudnicki) [inline app/models/requests/JourneyRequest.scala]: It can be a `val`, but it's okay :)

File: app/services/SubmissionService.scala [hunk 3]
```diff
@@ -37,55 +35,71 @@ import scala.util.{Failure, Success}
...
   exportsMetrics: ExportsMetrics
 ) {
 
   private val logger = Logger(this.getClass)
 
-  def submit(
-    exportsDeclaration: ExportsDeclaration,
-    legalDeclaration: LegalDeclaration
-  )(implicit request: JourneyRequest[_], hc: HeaderCarrier, ec: ExecutionContext): Future[Option[String]] = {
+  def submit(eori: String, exportsDeclaration: ExportsDeclaration, legalDeclaration: LegalDeclaration)(
```
Comment (bambuchaAdm) [inline app/services/SubmissionService.scala]: why eori as string ?

**Pull request merged**